src/utils/helper.ts

```diff
@@ -40,3 +40,8 @@ export const toKebabCase = (str: string) => {
   const match: string[] = (str && str.match(KEBAB_REGEX)) || [];
   return match.map((x: string) => x.toLowerCase()).join('-');
 };
+
+export const toPascalCase = (str: string) =>
```

Comment:
nice utils but since we have lodash installed I wonder if we could just do:

`startCase(str.toLower());`